enhancements/kubelet/master-bootstrap-credentials.md

```diff
@@ -0,0 +1,120 @@
+---
+title: master-bootstrap-credentials
+authors:
+  - "@deads2k"
+reviewers:
+  - "@rphillips"
+approvers:
+  - "@rphillips"
+creation-date: 2020-01-31
+last-updated: 2020-01-31
+status: implementable
+see-also:
+  - https://bugzilla.redhat.com/show_bug.cgi?id=1693951
+replaces:
+superseded-by:
+---
+
+# Master Bootstrap Credentials
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Open Questions [optional]
+
+## Summary
+
+Update the master bootstrap credentials used to request the initial CSRs for a kubelet.
+Requirements for such a credential:
+ 1. long lived.  It should not expire on short time frames.
+ 2. revokable.  We must be able to revoke this credential if it is compromised.
+
+## Motivation
+
+In order to scale masters horizontally, replace masters, and shutdown and restore clusters,
+we need to have master bootstrap credentials that last longer.
+On workers, we use service account credentials
+
+### Goals
+
+Make it possible to
+ 1. scale masters horizontally
+ 2. replace masters
+ 3. shutdown and restore clusters
```

Comment:
I think this is still conditioned similar to https://github.com/openshift/enhancements/pull/196/files#r373699897